### ISSUE TITLE
No Response object when logging at DEBUG level

### ISSUE THREAD

**Issue by jwallinder** [googleads/google-ads-python#498]
**Describe the bug:**
When we enable logging at DEBUG level ([according to documentation](https://developers.google.com/google-ads/api/docs/client-libs/python/logging#log_levels)) we can see the requests and responses printed to the log. But the response object does not contain any response body, it is `None`

This is specially important when we are asked by Google Ads API support to enable DEBUG logging and send in full request and response. Without a full response object we are declined help.

```
[2021-09-28 08:16:13 - DEBUG] Request
-------
Method: /google.ads.googleads.v8.services.GoogleAdsService/SearchStream
Host: googleads.googleapis.com
Headers: {
  "developer-token": "REDACTED",
  "login-customer-id": "<removed>",
  "x-goog-api-client": "gl-python/3.8.2 grpc/1.40.0 gax/1.31.2",
  "x-goog-request-params": "customer_id=<removed>"
}
Request: customer_id: "<removed>"
query: "SELECT customer.descriptive_name, campaign.advertising_channel_type, campaign.advertising_channel_sub_type, campaign.id, campaign.name, campaign.tracking_url_template, campaign.url_custom_parameters, campaign.final_url_suffix, segments.date, segments.sk_ad_network_conversion_value, metrics.sk_ad_network_conversions, customer.currency_code FROM campaign WHERE segments.date BETWEEN \'2021-01-01\' AND \'2021-01-01\'"


Response
-------
Headers: {
  "request-id": "<removed>"
}
Response: None
```

**Steps to Reproduce:**
Enable DEBUG logging
make a request

**Expected behavior:**
Response object NOT `None`

**Client library version and API version:**
Client library version: 12.0.0
Google Ads API version: v8

**Request/Response Logs:**

 
**Anything else we should know about your project / environment:**

**Comment by jwallinder** [comment]
This is how our log-config look like:
```
logg_config = {
    "disable_existing_loggers": False,
    "version": 1,
    "formatters": {
        "default_fmt": {
            "format": "[%(asctime)s - %(levelname)s] %(message).5000s",
            "datefmt": "%Y-%m-%d %H:%M:%S",
        }
    },
    "handlers": {
        "default_handler": {
            "class": "logging.StreamHandler",
            "formatter": "default_fmt",
        }
    },
    "loggers": {"": {"level": "DEBUG", "handlers": ["default_handler"]}},
}
```

**Comment by BenRKarl** [comment]
@jwallinder Can you confirm whether you get different results at other levels, i.e. `INFO` or `ERROR`? I believe that this is a known issue due to the way the request interceptors process streaming responses - the interceptors do not fire until the stream is exhausted so there is no data left to log. I believe this is going to take some improvements to the library, which I'm hoping to prioritize soon.

**Comment by jwallinder** [comment]
Hi @BenRKarl. Yes I kind of get a different result, nothing is printed out because the level of the message is DEBUG. I have updated the printout in the issue to include the date and level (full printout from the python lib). 
I hope you will prioritise this, we have blocked support issues from the Google Ads API support.

**Comment by BenRKarl** [comment]
@jwallinder that's what I'd expect. While we get this work gets prioritized, there are a few options for you to move forward with Support. First, is it possible to generate the same error you're receiving using `search` instead of `search_stream`? If not, you should still be able to retrieve the `request-id` from streaming logs, and the Support team can use that to look up the full error logs internally. Feel free to reference this Issue with them and let them know they can loop me into the conversation if needed.

**Comment by jwallinder** [comment]
@BenRKarl If we use `search` we get the response object printed out, as expected. 
We have also told Google Ads support to reach out to you if they need assistance in finding the response from the `request-id`

**Comment by BenRKarl** [comment]
This is fixed in version `15.1.0`
